src/Authoring/IInboundContext.cs

```diff
@@ -354,8 +354,14 @@ public interface IInboundContext : IHaveExpressionContext
     void Trace(TraceConfig config);
 
     /// <summary>
-    ///     TODO
+    /// TODO
     /// </summary>
     /// <param name="config"></param>
     void ValidateAzureAdToken(ValidateAzureAdTokenConfig config);
+
+    /// <summary>
+    /// TODO
```

Comment:
Let's not TODO customer facing docs

Reply:
For now, I want to move with the implementation. I will be revisiting that before the release.